### PR TITLE
Setup for funding on testnet 

### PR DESCRIPTION
**Pull request by svetomir**
This PR updates funding so Beekeeper can support multiple blockchains.
It also adds configuration for starting Bee nodes as a members of Bee Testnet.

### CODE REVIEW

File: cmd/beekeeper/cmd/fund.go [hunk 1]
```diff
@@ -33,19 +35,33 @@ beekeeper fund --addresses=0xf176839c150e52fe30e5c2b5c648465c6fdfa532,0xebe269e0
...
 				}
-				fmt.Printf("%s funded with %.2f ETH, transaction: %s\n", a, ethDeposit, tx)
-
+				// BZZ funding
 				bzzDeposit := c.globalConfig.GetFloat64(optionNameBzzDeposit)
-				tx, err = c.swapClient.SendBZZ(ctx, a, bzzDeposit)
-				if err != nil {
-					return fmt.Errorf("deposit bzz: %w", err)
+				if bzzDeposit > 0 {
+					tx, err := c.swapClient.SendGBZZ(ctx, a, bzzDeposit)
```
[the remark this answers is not in the document]
Comment (svetomir) [reply cmd/beekeeper/cmd/fund.go:50]: There is already a method SendBZZ, this is additional for Goerli.

File: config/testnet.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
+# clusters defines clusters Beekeeper works with
+# clusters may inherit it's configuration from already defined cluster and override specific fields from it
+clusters:
+  testnet:
+    _inherit: ""
+    name: bee
+    namespace: svetomir
```
[the remark this answers is not in the document]
Comment (svetomir) [reply config/testnet.yaml:7]: Changed

File: pkg/bee/nodegroup.go [hunk 3]
```diff
@@ -315,55 +314,84 @@ func (g *NodeGroup) DeleteNode(ctx context.Context, name string) (err error) {
...
+func (g *NodeGroup) Fund(ctx context.Context, name string, o FundingOptions) (err error) {
 	var a Addresses
-	retries := 5
-	for retries > 0 {
-		c, err := g.NodeClient(name)
-		if err != nil {
-			return err
+	if o.Eth > 0 || o.Bzz > 0 || o.GBzz > 0 {
+		retries := 5
+		for retries > 0 {
```
[the remark this answers is not in the document]
Comment (svetomir) [reply pkg/bee/nodegroup.go]: makes sense

File: cmd/beekeeper/cmd/fund.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (janos) [reply cmd/beekeeper/cmd/fund.go:50]: Yes, but SendGBZZ is used a few lines bellow to send the amount of gBzzDeposit. Here, the same SendGBZZ is used to send the bzzDeposit. Should SendGBZZ be used for sending both gBzzDeposit and bzzDeposit?

[the remark this answers is not in the document]
Comment (svetomir) [reply cmd/beekeeper/cmd/fund.go]: I see, thank you, good catch. Fixed

**Review by notanatol — APPROVED** [review]

**Review by janos — APPROVED** [review]
Great. Thanks! :)

**Pull request merged**